lib/optimizely/config_manager/http_project_config_manager.rb

```diff
@@ -197,7 +197,7 @@ def set_config(config)
         "Old revision number: #{previous_revision}. New revision number: #{@config.revision}.")
     end
 
-    def polling_interval(polling_interval)
+    def set_polling_interval(polling_interval)
```

Comment:
we can't really rename it.

Reply:
I don't see any issue in renaming. These are private methods of the class.